src/server/components/charts-engine/components/processor/chart-api-context.ts

```diff
@@ -43,12 +43,15 @@ export const getChartApiContext = (args: GetChartApiContextArgs): ChartApiContex
 
     const api: IChartEditor = {
         getSharedData: () => shared,
+        setSharedData: (override: Record<string, object>) => {
```

Comment:
Is this to replace the mutation? Will it be available to users as well?

Reply:
i removed it.